### PR TITLE
Rationalize async CallSettings. Fix copyright.

### PR DESCRIPTION
**Pull request by chrisdunelm**

### CODE REVIEW

File: src/Google.Datastore.V1Beta3/DatastoreClient.cs [hunk 1]
```diff
@@ -165,6 +163,23 @@ public virtual Datastore.IDatastoreClient GrpcClient
...
+        public virtual Task<LookupResponse> LookupAsync(
+            string projectId,
+            ReadOptions readOptions,
+            IEnumerable<Key> keys,
+            CancellationToken cancellationToken) => LookupAsync(
+                projectId,
+                readOptions,
+                keys,
+                new CallSettings { CancellationToken = cancellationToken });
+        /// <summary>
```
Comment (jskeet) [inline src/Google.Datastore.V1Beta3/DatastoreClient.cs]: Nit: Can we tweak the generator to add a blank line here?

Comment (chrisdunelm) [inline src/Google.Datastore.V1Beta3/DatastoreClient.cs]: Yes, thanks for spotting it

**Comment by chrisdunelm** [comment]
Done, PTAL

**Comment by jskeet** [comment]
LGTM.

**Pull request merged**